MC/bin/o2dpg_sim_workflow.py

```diff
@@ -994,6 +994,9 @@ def putConfigValues(listOfMainKeys=[], localCF = {}):
 
    tpcLocalCF={"DigiParams.maxOrbitsToDigitize" : str(orbitsPerTF), "DigiParams.seed" : str(TFSEED)}
 
+   # force TPC common mode correction in all cases to avoid issues the CMk values stored in the CCDB
+   tpcLocalCF['TPCEleParam.DigiMode'] = str(2) # 2 = o2::tpc::DigitzationMode::ZeroSuppressionCMCorr from TPCBase/ParameterElectronics.h
```

Comment:
so we just force this all the time, not just for some runs that are buggy?

Reply:
Well, for all other runs this is done by the default auto mode anyhow ...

Reply:
Did someone test that it actually works and fixes our problems? Then I can merge.